core/domain/platform_parameter_domain.py

```diff
@@ -339,7 +339,7 @@ def _evaluate_single_value(
         Raises:
             Exception. Given operator is not supported.
         """
-        if self._type in ['platform_type', 'browser_type'] and op != '=':
+        if self._type in {'platform_type', 'browser_type'} and op != '=':
```

Comment:
It doesn't matter too much, but typically we use tuples (..., ...) for this sort of thing.

Reply:
Updated